eslint.config.js

```diff
@@ -187,7 +187,9 @@ export default [
 				originaljQuery: false,
 				original$: false,
 				baseURL: false,
-				externalHost: false
+				externalHost: false,
+				Promise: false,
+				Symbol: false
```

Comment:
Please add these two directly above `trustedTypes` so that it's close to other built-ins. Also, the diff will be nicer.

Reply:
Hi...resolved this.